pre_commit/clientlib/validate_manifest.py

```diff
@@ -25,6 +25,13 @@ class InvalidManifestError(ValueError):
             'language_version': {'type': 'string', 'default': 'default'},
             'files': {'type': 'string'},
             'expected_return_value': {'type': 'number', 'default': 0},
+            'stages': {
+                'type': 'array',
+                'default': [],
```

Comment:
ah yeah I guess that's fair.  The tradeoff I see here is slightly easier to debug here but needs a test to keep it from regressing vs an extra branch in the other code.  I usually lean towards the first but this is fine as long as the code executes correctly.